sim/battle-actions.ts

```diff
@@ -749,8 +749,11 @@ export class BattleActions {
 		return undefined;
 	}
 	/** NOTE: used only for moves that target sides/fields rather than pokemon */
-	tryMoveHit(target: Pokemon, pokemon: Pokemon, move: ActiveMove): number | undefined | false | '' {
-		this.battle.setActiveMove(move, pokemon, target);
+	tryMoveHit(targetOrTargets: Pokemon | Pokemon[], pokemon: Pokemon, move: ActiveMove): number | undefined | false | '' {
```

Comment:
I think it's probably unnecessary to pass the array here. Just grab the array of foe sides if `move.target === 'foeSide'`.

Reply:
I guess we can worry about that later, though.